package.json

```diff
@@ -124,85 +306,309 @@
-          "tags": [
-            "telemetry"
-          ],
-          "scope": "window"
-        },
-        "kaoto.catalog.url": {
-          "type": "string",
-          "default": null,
-          "markdownDescription": "URL to a Kaoto catalog. For instance `https://raw.githubusercontent.com/KaotoIO/catalogs/main/catalogs/index.json`. Documentation to generate your own set of catalog is available [here](https://github.com/KaotoIO/kaoto/tree/main/packages/catalog-generator). It requires to reopen the Kaoto editors to be effective.",
-          "scope": "window"
-        },
-        "kaoto.nodeLabel": {
-          "type": "string",
-          "default": "description",
-          "markdownDescription": "Node label, which will be used for nodes in the canvas. Can be either `description` or `id`. If `description` is selected, it will be displayed only if it is available, otherwise `id` will be displayed by default. It requires to reopen the Kaoto editors to be effective.",
-          "enum": [
-            "description",
-            "id"
-          ],
-          "scope": "window"
-        },
-        "kaoto.nodeToolbarTrigger": {
-          "type": "string",
-          "default": "onHover",
-          "markdownDescription": "Choose when to open the Node toolbar. Can be either `onHover` or `onSelection`. If `onHover` is selected, the toolbar will be automatically open upon hovering a node, otherwise, it will be open when selecting a node. It requires to reopen the Kaoto editors to be effective.",
-          "enum": [
-            "onHover",
-            "onSelection"
-          ]
-        },
-        "kaoto.enableDragAndDrop": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "Control whether to enable drag and drop feature. It requires to reopen the Kaoto editors to be effective.",
-          "tags": [
-            "experimental"
-          ]
-        }
+    "submenus": [
+      {
+        "id": "kaoto.new.file",
+        "label": "New Camel File...",
+        "icon": "$(new-file)"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "camel-integrations-view",
+          "title": "Kaoto",
+          "icon": "icons/kaoto-icon.png"
+        }
+      ]
     },
+    "views": {
+      "camel-integrations-view": [
+        {
+          "id": "kaoto.integrations",
+          "name": "Integrations",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.deployments",
+          "name": "Deployments",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.openapi",
+          "name": "Open API",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.datamappings",
+          "name": "Data Mappings",
+          "contextualTitle": "Kaoto"
+        },
```

Comment:
given that for now we are not doing anything except displaying them, I think we could start with them collapsed. It is also some sections that won't always have something in it.